assemblies/data-access-plugin/src/main/resources/resources/web/pentaho-ajax.js

```diff
@@ -12,327 +12,304 @@
 * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 * See the GNU Lesser General Public License for more details.
 *
-* Copyright (c) 2002-2017 Hitachi Vantara..  All rights reserved.
+* Copyright (c) 2002 - 2019 Hitachi Vantara..  All rights reserved.
 */
 
+// Project: data-access
+
+/* globals CONTEXT_PATH, FULL_QUALIFIED_URL */
+
 // see http://developer.mozilla.org/en/docs/AJAX:Getting_Started for other values
 var COMPLETE = 4;
+
 // see http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html for other values
 var STATUS_OK = 200;
 var STATUS_UNAUTHORIZED = 401;
 var STATUS_NOT_FOUND = 404;
+
 var base = '';
 
- document.write("<!-- webcontext.js is written by PentahoWebContextFilter. Content of this file contains values of CONTEXT_PATH and FULLY_QUALIFIED_SERVER_URL --> \n"); 
- document.write("<script type='text/javascript' src='webcontext.js'></script>"); 
+document.write("<!-- webcontext.js is written by PentahoWebContextFilter. Content of this file contains values of CONTEXT_PATH and FULLY_QUALIFIED_SERVER_URL --> \n");
+document.write("<script type='text/javascript' src='webcontext.js'></script>");
 
```

Comment:
This seems to be the only relevant change of this file, in relation to the common-ui's version.
We can probably still load the central version and only do this here.